educator/src/Dscp/Educator/Web/Educator/Queries.hs

```diff
@@ -270,16 +263,18 @@ educatorPostGrade subH grade = do
 -- and generates a PDF certificate with embedded FairCV describing that block.
 educatorAddCertificate
     :: MonadEducatorWeb ctx m
-    => CertificateFullInfo -> m ()
+    => CertificateFullInfo -> m PDFBody
 educatorAddCertificate cert = do
-    pdfLatexPath <- view (lensOf @Pdf.LatexPath)
-    pdfResPath <- view (lensOf @Pdf.ResourcePath)
+    pdfLatexPath    <- view (lensOf @Pdf.LatexPath)
+    pdfResPath      <- view (lensOf @Pdf.ResourcePath)
     downloadBaseUrl <- view (lensOf @Pdf.DownloadBaseUrl)
-    certificateIssuerInfo <- getCertificateIssuerInfo
```

Comment:
Rebase artifacts? That's what @pasqu4le has added just recently (and implementation just below).

Reply:
What about this?